pom.xml

```diff
@@ -36,34 +36,43 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>subversion</artifactId>
-            <version>1.38</version>
-            <optional>true</optional>
```

Comment:
The dependency is used in compilation time so I would assume it should not be optional. I've looked at the commit that made the dependency optional, and it was the only change between versions 2.5 and 2.6, so there may be other implications I have not thought of. For that reason I have made it optional again :) Thanks for bringing it up!